src/components/NavigationMailbox.vue

```diff
@@ -579,6 +579,14 @@ export default {
 							.dispatch('deleteMailbox', { mailbox: this.mailbox })
 							.then(() => {
 								logger.info(`mailbox ${id} deleted`)
+								if (this.$route.params.mailboxId === this.mailbox.databaseId) {
```

Comment:
This can never be truthy. `this.$route.params.mailboxId` is a string, `this.mailbox.databaseId` is a Number.

Reply:
It is an int before the page load. After a page load it becomes a string.